metabase/metabase
# WIP - 29598 csv status toast

Closes https://github.com/metabase/metabase/issues/29598

### Description
Adds redux state to handle showing the alert in the bottom right corner while uploading a CSV, as well as generalizes the Status Listings. Setup should handle multiple concurrent uploads happening at once, and each collection should have it's own listing. Currently the upload is mocked to take 6 seconds.

### How to verify

1. Go to any collection, and "upload" a csv
2. Alert should appear with the correct collection name in the header.

### Demo
![chrome_nA9AxjzGgb](https://user-images.githubusercontent.com/1328979/229539466-36b65926-b211-422d-8441-3cf3fe0ae5ae.gif)

### Checklist


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/29757)
